### PR TITLE
TIS-399 Package.swift Enable SPM

### PR DESCRIPTION
**Pull request by danilobecke**

### CODE REVIEW

**Review by vipinsainitech — APPROVED** [review]

**Review by vipinsainitech — APPROVED** [review]

**Comment by RGG-jayoung** [comment]
Has there been any progress on adding SPM support?

**Comment by rogerluan** [comment]
Ping @mooseriskified 😇 Perhaps this could be merged in before the next lib release which looks like it's just around the corner? 🙏

**Comment by rogerluan** [comment]
Ping @mooseriskified

**Comment by mooseriskified** [comment]
@rogerluan Thank you - ill add it this week

**Comment by alfogrillo** [comment]
Sorry I just noticed this PR after I opened a similar one here: https://github.com/Riskified/ios_sdk/pull/14
I just would suggest to lower the minimum iOS target to 9.0 if you don't see any problem with that.

**Comment by username0x0a** [comment]
Well, the framework is built with `9.0` deployment, but the `xcframework` bundle only contains `arm64` slice for iOS devices 🤔 so it will only work on 64-bit ones. Keeping that in mind, I'd better update the whole framework to `11.0` deployment target which effectively dropped `armv7` architecture for good.

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// swift-tools-version:5.5
+
+import PackageDescription
+
+let package = Package(
+    name: "RiskifiedBeacon",
+    platforms: [ .iOS(.v13) ],
```
Comment (rogerluan) [inline Package.swift:7]:
```suggestion
    platforms: [ .iOS(.v11) ],
```

**Comment by Killectro** [comment]
Is there any update on adding support for SPM? We would really love to use it via SPM rather than Cocoapods or manual integration

**Comment by rogerluan** [comment]
Ping @mooseriskified

**Pull request closed without merging**